patches/0008-flatpak-Adjust-paths-for-the-sandbox.patch

```diff
@@ -80,7 +80,21 @@ index 26dc3fc1cafa8..8f26e588b4e63 100644
  #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
        cur = base::FilePath(FILE_PATH_LITERAL("/etc/opt/chrome/policies"));
  #else
-@@ -521,8 +533,8 @@ bool PathProvider(int key, base::FilePath* result) {
+@@ -511,7 +523,12 @@ bool PathProvider(int key, base::FilePath* result) {
+ #endif
+ #if defined(OS_LINUX) || defined(OS_CHROMEOS)
+     case chrome::DIR_STANDALONE_EXTERNAL_EXTENSIONS: {
+-      cur = base::FilePath(kFilepathSinglePrefExtensions);
++      if (sandbox::FlatpakSandbox::GetInstance()->GetSandboxLevel() >
++          sandbox::FlatpakSandbox::SandboxLevel::kNone) {
++        cur = base::FilePath(FILE_PATH_LITERAL("/app/extensions"));
```

Comment:
Looks like we also need to adjust the other paths here (e.g. `/app/policies`) with the changes to the extension points dirs.